pyproject.toml

```diff
@@ -53,6 +53,7 @@ tests = [
   "pytest-mpl",
   "pytest-httpserver",
   "pytest-filter-subpackage",
+  "pytest-codspeed",
```

Comment:
As this is really only a CI dependency I would put it in the github actions workflow and not here.